src/LanguageServer/IdePurescript/CodeActions.purs

```diff
@@ -54,7 +54,7 @@ getActions documents settings (ServerState { diagnostics, conn: Just conn }) { t
 
     fixAllCommand text rebuildErrors = if length replacements > 0 then [ replaceAllSuggestions text docUri replacements ] else [ ]
       where
-      replacements = mapMaybe getReplacementRange rebuildErrors
+      replacements = nubByEq eq $ mapMaybe getReplacementRange rebuildErrors
```

Comment:
This is the only non-whitespace change.

Reply:
Somehow suggestions were appearing multiple times which lead to none being executed (I don't know why). Deduplicating this helped. We could also use `nub` but there was no `Ord` instance for `Range` and I tried to make the smallest change possible.

Reply:
Well hard to see how this could go wrong, at worst no change...

Reply:
That's what I thought, too. I was thinking we could merge this in and then see if we can still reproduce the problem.